composer.json

```diff
@@ -8,20 +8,24 @@
     "2-factor",
     "authentication",
     "module",
-    "security"
+    "security",
+    "TOTP",
+    "Authy",
+    "GoogleAuthenticator"
   ],
   "authors": [
     {
       "name": "Elliot Sawyer"
     }
   ],
   "require": {
-    "firesphere/bootstrapmfa": "dev-master#f0fc8eb861b7e5dca3398dd9f59a382f5cdf5a23",
+    "firesphere/bootstrapmfa": "dev-extension-hook-before-mfa",
     "endroid/qr-code": "3.2.12",
-    "lfkeitel/phptotp": "^1.0",
+    "spomky-labs/otphp": "^9.1",
     "silverstripe/framework": "^4"
   },
   "require-dev": {
+    "roave/security-advisories": "dev-master",
```

Comment:
This isn't a standardised dependency so we shouldn't add it yet (see https://github.com/silverstripe/silverstripe-framework/issues/8548)